lib/sms/Kconfig

```
@@ -6,14 +6,22 @@
 
 menuconfig SMS
 	bool "SMS subscriber"
-	imply AT_CMD
-	imply AT_CMD_PARSER
-	imply AT_NOTIF
+	select AT_CMD_PARSER
+	select AT_NOTIF
 	help
 	  A library for managing SMS subscriptions.
 
 if SMS
 
+config SMS_AT_CMD
+	bool "SMS uses AT Command library"
```

I believe this should be promptless.
We don't want end-users to disable this, do we ?.

```suggestion
	bool
```

---

Yep, users shouldn't touch it. And I guess we don't need to add specific references to this in the SMS library documentation.